src/drivers/distance_sensor/ll40ls/LidarLite.h

```diff
@@ -45,7 +45,8 @@
 
 /* Device limits */
 #define LL40LS_MIN_DISTANCE (0.05f)
-#define LL40LS_MAX_DISTANCE (25.00f)
+#define LL40LS_MAX_DISTANCE (35.00f)
+#define LL40LS_MAX_DISTANCE_OLD (25.00f)
```

Comment:
Can we refer to lidar lite versions instead of just "old"?

Reply:
Ok. I renamed them V1 and V3

Reply:
Oh, I thought the distinction was V3 and V3HP (not that I have either of these at the moment).

Reply:
In theory the driver was supposed to support V1 and V3 before this pr. The main difference was in the i2c slave address. Now I added V3HP and slightly changed how to probe v3 so the two probe options are for v1 and v3/v3HP. For these two cases I probe different registers because the HP doesn't respond to the hw_version and sw_version registers. So for v1 I still check these two registers while for v3/v3HP I check the unit_id register instead.